frontend/src/features/public-form/PublicFormPage.stories.tsx

```diff
@@ -863,3 +863,18 @@ ThankYouPage.play = async ({ canvasElement }) => {
     },
   )
 }
+
+export const WithRespondentCopy = Template.bind({})
+WithRespondentCopy.parameters = {
+  msw: [
```

Comment:
Praise: great work with mocking the FE for chromatic stories, this will ensure respondent copy is well tested for the rest of time!